.gitignore

```diff
@@ -0,0 +1,4 @@
+elm-stuff
+repl-temp-*
```

Comment:
Hi. This is not really relevant to the PR, but I'm curious what is your setup with these `repl-temp-` files/directories. I often use files like `tmp_repl.txt` with two horizontally split windows where I send lines of code with a shortcut to the bottom window. Are you using something similar?
![image](https://user-images.githubusercontent.com/2153100/220325334-191da860-de9c-4d8e-b0ea-a18eff8022a3.png)